incalmo/c2server/state_store.py

```diff
@@ -1,55 +1,40 @@
 import json
 import os
+import sqlite3
 from typing import Any, Optional
 
-from redis import Redis
-
-
 class StateStore:
-    _redis_client: Optional[Redis] = None
-    _memory_cache: dict[str, Any] = {"environment:hosts": []}
-
-    @classmethod
-    def _get_redis_client(cls) -> Optional[Redis]:
-        if cls._redis_client is not None:
-            return cls._redis_client
-
-        # Prefer explicit state store URL, then Celery backend/broker, then default
-        redis_url = (
-            os.environ.get("STATE_REDIS_URL")
-            or os.environ.get("result_backend")
-            or os.environ.get("broker_url")
-            or "redis://localhost:6379/0"
-        )
-
-        client = Redis.from_url(redis_url, decode_responses=True)
-        # Validate connection
-        client.ping()
-        cls._redis_client = client
-        return cls._redis_client
+    TABLE_NAME = "environment"
+    DB_PATH = "state_store.db"
```

Comment:
Every time you start Incalmo their should be a fresh DB. Can you add this to the gitignore, and maybe delete or reset this file on start?